q2-structure/proposal-by-xdg.md

```diff
@@ -0,0 +1,220 @@
+
+They alone elect the Steering Council and have the power to remove Steering
+Council members.
+
+In concert with the Steering Council, they manage their own membership and
+have the power to modify the Perl Rules of Governance.
+
+The Core Team do not have any authority over parts of the Perl ecosystem
+unrelated to developing and releasing the language itself.
+
+### Membership, Inactivity, and Removal
+
+The initial Core Team members will be specified in the first draft of the Perl
+Rules of Governance.
+
+Core Team members who have stopped contributing are encouraged to declare
+themselves as "inactive". Inactive members do not participate in votes.
+Inactive members may declare themselves active at any time, except when a vote
+has been proposed and is not concluded.  Eligibility to vote will be
+determined by the Vote Administrator.
+
+Core Team Members may also resign their position at any time.
+
+Any Core Team member may nominate someone to be added to the Core Team by
+sending the nomination to the Steering Council.  The Steering Council must
+approve or reject the nomination.  If approved, the Steering Council will
+organize a Membership Change Vote to ratify the addition.
+
+In exceptional circumstances, it may be necessary to remove someone from the
+Core Team against their will, such as for flagrant or repeated violations of a
+Code of Conduct.  Any Core Team member may send a recall request to the
+Steering Council naming the individual to be removed.  The Steering Council
+must approve or reject the recall request.  If approved, the Steering Council
+will organize a Membership Change vote to ratify the removal.
+
+If the removed member is also on the Steering Council, then they are removed
+from the Steering Council as well.
+
+### No Confidence in the Steering Council
+
+The Core Team may remove either a single Steering Council member or the entire
+Steering Council via a No Confidence Vote.
+
+A No Confidence Vote is triggered when a Core Team member calls for one
+publicly on an appropriate project communication channel, and another Core
+Team member seconds the proposal.
+
+If a No Confidence Vote removes all Steering Council members, the Vote
+Administrator of the No Confidence Vote will then administer a Term Election
+to select a new Steering Council.
+
+### Amending Perl Rules of Governance
+
+Any Core Team member may propose amending the Perl Rules of Governance by
+sending a proposal to the Steering Council.  The Steering Council must decide
+to approve or reject the proposal.  If approved, the Steering Council will
+administer an Amendment Vote.
+
+## The Perl Steering Council
+
+The Perl Steering Council is a 5-person committee, elected by the Perl Core
+Team.  Candidates are not required to be members of the Core Team.  Non-member
+candidates are added to the Core Team if elected as if by a Membership Change
+Vote.
+
+### Powers
+
+The Steering Council has broad authority to make decisions about the
```

Comment:
Another case where I think PEP 13 is significantly clearer.  The Mandate and Powers sections are way more clear on what "broad authority" means.  It also makes it a little clearer how it relates to the Core Team.  

It also contains this beautiful paragraph:

```
The council should look for ways to use these powers as little as possible. Instead of voting, it's better to seek consensus. Instead of ruling on individual PEPs, it's better to define a standard process for PEP decision making (for example, by accepting one of the other 801x series of PEPs). It's better to establish a Code of Conduct committee than to rule on individual cases. And so on.
```

I think that nicely frames the guiding principles of their steering council.

Reply:
I love that paragraph, but it's non-binding and thus not structural.  The [Q1 proposal](https://github.com/Perl/governance/blob/main/q1-mandate-and-powers/proposal-by-xdg.md) we approved contains a lot of guidance that I didn't want to repeat here.  I only put in this section to divide the powers between Core/Council in a way that I was told not to do in Q1.

As I think of Q2 as "requirements" for the final doc, I'm going to leave it as is, but assuming this proposal passes and I draft a final document, I intend to include clarifiying guidance and examples.

Reply:
Maybe it would be clearer to drop all the "powers" stuff from this doc entirely then?  To avoid a "we already decided that in Q2 and now you want to change it in Q3" discussion.  Unless that's actually ok...

Reply:
Added examples and something like that paragraph to the revision I just pushed.